02.calendar/calender.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+#引数yで年、mで月を渡す
+options = ARGV.getopts("", "y:#{Date.today.year}", "m:#{Date.today.month}")
+
+#yearとmonth変数それぞれにコマンドで渡した値を格納
+year = options["y"].to_i
+month = options["m"].to_i
+
+#月初と月末の値を取得し日数を求める
+start = Date.new(year,month,1)
```

Comment:
これも大したことではないですしRubocopを使えば機械的に指摘されるとは思いますが、引数が複数ある場合`,`の後に半角スペースを開けた方が良いです。
```suggestion
start = Date.new(year, month, 1)
```

Reply:
引数後の半角スペースも以後気をつけます！